Assets/VRTK/Scripts/Interactions/VRTK_ControllerActions.cs

```diff
@@ -63,6 +65,9 @@ public struct ControllerActionsEventArgs
     /// </example>
     public class VRTK_ControllerActions : MonoBehaviour
     {
+        [Tooltip("The controller who's events will be propagated.")]
```

Comment:
Seems like this is copied over from `VRTK_ControllerEvents`.
The tooltip should probably read something along the lines of "The hand of the controller to provide the helper methods for. If this is set to `None` the game object this script is attached to will be used.".